en/starter/generator/getting_started.md

```diff
@@ -0,0 +1,153 @@
+
+Press **Enter** to accept the default plural form (`CoffeeShops`):
+
+    [?] Custom plural form (used to build REST URL): (CoffeeShops)
+
+Every model has properties.  Right now, you’re going to define one property, `name`, for the CoffeeShop model:
+
+    Let's add some CoffeeShop properties now.
+
+    Enter an empty property name when done.
+    [?] Enter the property name: name
+
+Select `string` as the property type (press **Enter**, as `string` is the default choice):
+
+    [?] Property type: (Use arrow keys)
+    > string
+    number
+    boolean
+    object
+    array
+
+Each property can be optional or required. Enter `y` to make name required:
+
+    [?] Required? (y/N)
+
+Then you'll be prompted to enter a default value for the property; press **Enter** for no default value:
+
+    [?] Default? \[y/N\]:
+
+Then, you'll be prompted to add another property. Follow the prompts to add a required property named `city` which does not have a default value.
+
+    ? Enter the property name: city
+    ? Property type: string
+    ? Required? Yes
+    ? Default? \[y/N\]:
+
+End the model creation process by pressing **Enter** when prompted for the name of the next property.
+
+The model generator creates a file in the application’s `models` directory that defines the model: `CoffeeShop.json`
+
+---
+
+## Check out the project structure
+
+For all the details of the Swift Server Generator application structure, see [Project layout reference](project_layout_reference.md).
+
+---
+
+## Run the application
+
+Start the application:
+
+    $ .build/debug/swiftserver-getting-started
+
+To confirm that the project is running locally, open the URL
+`http://localhost:8090` in your browser. You'll see something like this:
+
+    Welcome to Kitura
+    Your Kitura based server is up and running!
+
+Through a set of simple steps using the Swift Server Generator tool, you’ve created a CoffeeShop model, specified its properties and then exposed it through REST.
+
+Models that are created by following this tutorial have a standard set of create, read, update and delete operations defined. See [Exposing Swift Server Generator models over REST](exposing_ssg_models_over_rest.md) for details. You can test any of the API endpoints by using **`curl`** commands. For example, to display all the model instances in the `swiftserver-getting-started` project, enter the following command (the `-k` option avoids certificate errors):
+
+    curl --request GET \\
+    --url 'https://localhost:8090/api/CoffeeShops' \\
+    --header 'accept: application/json' \\
+    --header 'content-type: application/json' \\
+    --header 'x-ibm-client-id: default' \\
+    --header 'x-ibm-client-secret: SECRET' -k

```

Comment:
This curl command is using the wrong protocol (should be http) and contains a lot of extra stuff that is not needed, especially the last two parameters which are API Connect specific and so not appropriate for this page. Should be:
```
curl http://localhost:8090/api/CoffeeShops
```